### DISCUSSION TITLE
Inconsistent GO results for sus scrofa (pig) gene sets

### DISCUSSION THREAD

**Discussion by daniel-palmer** [geneontology/helpdesk#454] [Q&A]
Hello! I have been using GO to analyze differentially expressed gene sets from a study using sus scrofa (pig) samples. Earlier this summer, I was able to enrich my gene set of 800+ genes for over 600 biological process terms with statistical significance. However, when I re-ran this analysis this week, I was unable to get any significant results. The differentially expressed genes are being affected by interleukin-1-mediated inflammation, which is associated with drastic changes in cell behavior, so it is unexpected to have no significant results. Any suggestions for what might have changed, or what I might do to resolve the problem? Thank you!

**Comment by suzialeksander** [comment]
Hi @daniel-palmer, sorry to hear that you're getting unexpected results.  We are resolving a bug (https://github.com/geneontology/helpdesk/issues/455), did you get an error message of any kind? Also, can you provide the URL(s) you are using, plus a small sample list so we can see if we can recreate this. 

We are constantly updating the ontology, and sometimes upstream changes have unintended effects. @pgaudet might be able to comment on if this could be an ontology problem.

**Reply by daniel-palmer** [reply]
Thank you for the prompt response! I am using https://geneontology.org/ . I did not get an error message; the analysis ran immediately, but did not return any significant results. I've attached an excel spreadsheet with a sheet showing my results on 6/30/23, compared to the second sheet showing my results 9/28/23, as well as a third sheet with the list of Ensembl gene codes I entered for both analyses. Does this help?

[PIEZO1 3hr GO.xlsx](https://github.com/geneontology/helpdesk/files/12751479/PIEZO1.3hr.GO.xlsx)

**Comment by pgaudet** [comment]
Hi @daniel-palmer 

Unfortunately the EnsEMBL team (from which most pig data is derived) seems to update their production pipeline and made changes to many genomes lately as well. I suggest you contact EnsEMBL directly and see if they can help understand what is happening, and which version is more reliable. 

Thanks, 

Pascale

**Reply by daniel-palmer** [reply]
Thank you, I will reach out to them! Is there a way to reference a previous version of the pig data, if EnsEMBL says the newer version is less reliable?

**Comment by pgaudet** [comment]
In GO you can use data from a previous release, available here: 
http://release.geneontology.org/

We highly recommend that you cite the  release date and/or version number of the data, preferably both; e.g. “2023-01-01” and “10.5281/zenodo.7504797, as described on the GO website: 

https://geneontology.org/docs/go-citation-policy/

Thanks, Pascale

**Reply by daniel-palmer** [reply]
Thank you! I am very new to this, so this may be an ignorant question, but how do I use the data? I can see it in the archive, but is there some way to select this version through the gene ontology resource/Panther settings?

**Comment by pgaudet** [comment]
@daniel-palmer 
For Panther questions, please contact hem directly: 
[pantherfeedback@yahoo.com](mailto:pantherfeedback@yahoo.com).

Please don't hesitate to ask if you have other questions, we'll do your best to help. 

Pascale
